save-cloud-common/src/jvmMain/kotlin/com/saveourtool/save/entities/Git.kt

```diff
@@ -21,11 +21,11 @@ class Git(
     @ManyToOne
     @JoinColumn(name = "organization_id")
     var organization: Organization,
-) : BaseEntity() {
+) : BaseEntityWithDto<GitDto>() {
     /**
      * @return git dto
      */
-    fun toDto() = GitDto(
+    override fun toDto() = GitDto(
         url = url,
         username = username,
         password = password,
```

Comment:
Do we always send all the credentials inside `GitDto`?

Reply:
yes

Reply:
Do we have an issue for that?

Reply:
Probably, but not in this PR )

Reply:
it's required more changed, also need to revise it additionally: we need to get somehow credential from backend to preprocessor. also access has only authorized users ~and only users who has access to the test suites source.~
We need to implement role managements for test suites source. It's not a scope of this task, but can add checking access to `Organization`

Reply:
Let's at least add **new** issue and move on :)

Reply:
actually, currently it can break public tests. we need to introduce `PermissionEvaluator` for `TestSuitesSource` and then we can rework it

Reply:
> Let's at least add **new** issue and move on :)

created: https://github.com/saveourtool/save-cloud/issues/1219